domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/response/SampleResponse.java

```diff
@@ -1,45 +1,24 @@
-package uk.gov.hmcts.cmc.domain.models.sampledata;
+package uk.gov.hmcts.cmc.domain.models.sampledata.response;
 
-import uk.gov.hmcts.cmc.domain.models.FullDefenceResponse;
+import uk.gov.hmcts.cmc.domain.models.PartAdmissionResponse;
 import uk.gov.hmcts.cmc.domain.models.Response;
 import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
 import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleParty;
 
 public abstract class SampleResponse<T extends SampleResponse<T>> {
 
-    public static class FullDefence extends SampleResponse<FullDefence> {
-        private FullDefenceResponse.DefenceType defenceType = FullDefenceResponse.DefenceType.DISPUTE;
-        private String defence = "defence string";
-
-        public static FullDefence builder() {
-            return new FullDefence();
-        }
-
-        public FullDefence withDefenceType(FullDefenceResponse.DefenceType defenceType) {
-            this.defenceType = defenceType;
-            return this;
-        }
-
-        public FullDefence withDefence(String defence) {
-            this.defence = defence;
-            return this;
-        }
-
-        public FullDefenceResponse build() {
-            return new FullDefenceResponse(
-                freeMediationOption, moreTimeNeededOption, defendantDetails, statementOfTruth,
-                defenceType, defence
-            );
-        }
+    public static PartAdmissionResponse validPartAdmissionDefaults() {
+        return SamplePartAdmissionResponse.builder().build();
     }
 
     protected Response.FreeMediationOption freeMediationOption = Response.FreeMediationOption.YES;
     protected Response.MoreTimeNeededOption moreTimeNeededOption = Response.MoreTimeNeededOption.YES;
     protected Party defendantDetails = SampleParty.builder().withRepresentative(null).individual();
     protected StatementOfTruth statementOfTruth;
 
-    public static FullDefenceResponse validDefaults() {
-        return FullDefence.builder().build();
+    public static Response validDefaults() {
```

Comment:
Why not to return `FullDefenceResponse` type? Name should also suggest what sort of response is being build here. I am asking for `validDefence` and `validPartAdmission` methods.

Reply:
Would the following be suitable?

    public static Response validDefence() {
        return SampleFullDefenceResponse.validDefence();
    }

Reply:
@rattur I meant just changing return type to more concrete type e.g. `FullDefenceResponse` instead of generic `Response`